src/handlers/resourceValidator.ts

```diff
@@ -129,7 +130,7 @@ export class ResourceValidator implements IOrchestratable {
             Logger.Debug(`- ${key} = ${configSettings.properties[key]}`);
         }
 
-        const result: FunctionSkuConstant = FunctionSkuUtil.FromString(configSettings.properties.sku);
+        const result: FunctionSkuConstant = (!!configSettings.properties.sku) ? FunctionSkuUtil.FromString(configSettings.properties.sku) : undefined;
```

Comment:
We still want the `result` here to be `FunctionSkuConstant` instead of undefined. In the `k4apps` scenario, will `configSettings.properties.sku` become an empty string? If so, please add an `Undefined` field in https://github.com/Azure/functions-action/blob/master/src/constants/function_sku.ts#L3, and add the conversion logic in the FunctionSkuUtil.FromString() function.